tests/markdown/variables.test.js

```diff
@@ -402,4 +402,10 @@ describe('Variable names that are subsets of other names', ()=>{
 		const rendered = Markdown.render(source).trimReturns();
 		expect(rendered).toBe('<p>14</p>');
 	});
+});
+
+describe('Don\'t Eat all the parentheticals!', ()=>{
+	const source='\n|  title 1  | title 2 | title 3 | title 4|\n|-----------|---------|---------|--------|\n|[foo](bar) |  Ipsum  |    )    |   )    |\n';
+	const rendered = Markdown.render(source).trimReturns();
+	expect(rendered).toBe('<table><thead><tr><th>title 1</th><th>title 2</th><th>title 3</th><th>title 4</th></tr></thead><tbody><tr><td><a href=\"bar\">foo</a></td><td>Ipsum</td><td>)</td><td>)</td></tr></tbody></table>');
```

Comment:
Tests need to be inside an `it()` call. `describe()` is a grouping that contains the individual `it()` tests, but the `describe()` isn't a test itself.